client/app/scripts/reducers/root.js

```diff
@@ -721,6 +724,34 @@ export function rootReducer(state = initialState, action) {
       return state.set('showingTroubleshootingMenu', !state.get('showingTroubleshootingMenu'));
     }
 
+    case ActionTypes.TOGGLE_CONTRAST_MODE: {
+      const modules = [
+        require.resolve('../../styles/main.scss'),
+        require.resolve('../../styles/contrast.scss')
+      ];
+      // Bust the webpack require cache to for a re-download of the stylesheets
+      modules.forEach((i) => {
+        const children = require.cache[i] ? require.cache[i].children : [];
+        children.forEach((c) => {
+          delete require.cache[c];
+        });
+        delete require.cache[i];
+      });
+
+      const contrastEnabled = state.get('contrastMode');
+      if (!contrastEnabled || action.enabled) {
```

Comment:
Yes that would work, so long as the user of the API call explicitly passes in `true` to the `toggleContrastMode`. My idea was that there are two scenarios: 
1. The user explicitly wants contrast mode
2. The user wants to toggle the between the two states, and doesn't necessarily care what the current state is. So calling `toggleContrastMode()` without an arg would switch from one to the other.

But then I realized that's stupid, so I simplified it with your recommendation :)